common/prototypes.h

```diff
@@ -259,6 +259,12 @@ uint32_t
 TDNFJoinPath(
     char **ppszPath, ...);
 
+uint32_t
+TDNFReadFileToStringArray(
+    const char *pszFile,
+    char ***pppszArray
```

Comment:
Can we use a double pointer instead?

Reply:
I don't see an issue with triple pointers.